src/web/types/gender.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Represents the gender of a student.
+ */
+export enum Gender {
+  MALE,
```

Comment:
Use string representative of enum.

e.g.
```
export enum Gender {
    MALE = "MALE"
}
```

so that in the runtime, the enum is a string (not a number and cannot be a number as backup give a string). (Note that there is no enum in JavaScript!)

Reply:
Yep, i already updated it in [this PR](https://github.com/TEAMMATES/teammates/commit/991995d9bdc7bb63945ba602d09d713eeca85806)

Reply:
So the lesson here is do not have two commits modifying the same place when you organise your commits task by task.

In this case, all final changes to `gender.ts` should contain in your first commit.

Reply:
Will rebase commits to fit this structure in the future